test/adapter.js

```diff
@@ -32,7 +29,12 @@ function testFixture(description, filename, subDir = true, generateSourceMap = f
       // Invoke with the env var GENERATE set to regenerate the fixtures.
       if (process.env.GENERATE) {
         expected = output.toRefract();
-        fs.writeFileSync(path.join(__dirname, expectedName), JSON.stringify(expected, null, 2), 'utf8');
+
+        if (generateSourceMap) {
```

Comment:
I'm not sure this makes sense. Maybe we should just remove the `GENERATE` option now? Really it should belong in the zoo repo instead.